Eastern-Research-Group/mywaterway
# Feature/156 stream gages to monitoring

## Related Issues:
* https://jira.epa.gov/browse/HMW-156

## Main Changes:
* Added USGS Stream Gages to the monitoring panel. 

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/monitoring
2. Verify the tab level switches work
3. Turn off the "Current Water Conditions" switch
4. Go to the 

## TODO:
- [x] I would like to have some discussion with EPA about what else we could do on the stream gages tab. It would be nice to add something to differentiate the stream gages data on the monitoring panel from the same data on the overview panel. I was thinking about maybe adding filters but I'm not sure how valuable that would be. It seems like the only thing we could feasibly filter on would be the "Water Type" attribute.
